### ISSUE TITLE
have you future plan to upgrade this library in ios swift ?

### ISSUE THREAD

**Issue by Sandeep155Vibhuti** [kadiks/react-native-itunes#28]
I have appreciated your work for this module @kadiks,
have this Library can support the audio Visualizer in iOS or
have you future plan to update this library in iOS swift?

**Comment by Nitish157** [comment]
Yes can you update it I am also facing same issue or can you add equalizer to this library?
this library could become much useful if they have equalizer or visualizer in it.

**Comment by MaxJadav** [comment]
Is there any updated library available for sync downloaded podcast from apple podcast app?

**Comment by kadiks** [comment]
Hi guys,

My career is web based and I'm not using react native anymore. So unless someone wants to take over the project, I'm going to archive this repo.

If you want to take over, contact me at: me@jenaiccambre.com
